Feature landing page

brandondykun opened the pull request:
Initial layout is complete. The text needs work but the initial layout is there. The page looks a little plain, so maybe we can come up with a cool background svg for the right side to add a little more flair.

Commit:
Added main landing page section.

Commit:
Merge branch 'develop' into feature-landing-page

Commit:
fix merge conflicts

Commit:
Add starter style to h1

RWransky commented on app/page.tsx:
```diff
@@ -1,7 +1,37 @@
-      <section id="about">
+    <main className="flex flex-col p-6 sm:p-12 flex-1">
+      <section className="min-h-[calc(100vh-80px)] sm:min-h-[calc(100vh-104px)] flex flex-col justify-center sm:pb-24 md:pl-6">
+        <h1 className="text-4xl sm:text-5xl lg:text-6xl text-center text-neutral-800 sm:text-left pb-8">
```
I am conflicted on this one. Technically a webpage should only ever have one `h1` tag and with my quick merge conflict fixes, I have introduced two...

I am wondering since this is the company name if it would be more appropriate to have it be the image of our logo rather than an `h1` text element?

RWransky commented on components/ui/button.tsx:
```diff
@@ -0,0 +1,59 @@
+  asChild?: boolean;
+}
+
+const Button = React.forwardRef<HTMLButtonElement, ButtonProps>(
```
Great button class! ✨

RWransky commented on app/page.tsx:
```diff
@@ -1,7 +1,37 @@
+            <Link href="/contact">Contact Us</Link>
+          </Button>
+          <Button variant="outline" asChild className="w-40">
+            <Link href="/#about">About Us</Link>
```
I am also conflicted a little here too with just this being a vanilla `Link`. Vanilla `Link` should be reserved for true route transitions imo, not in-page navigation. It feels a little too abrupt without a scroll or something.

Could you please convert to this `ScrollLink` [here](https://reacthustle.com/blog/nextjs-scroll-to-element)? It'll be handy in other pages too including that as a reusable component.

RWransky commented on app/page.tsx:
```diff
@@ -1,7 +1,37 @@
+      </section>
+      <h1
+        id="about"
+        className="text-xl md:text-2xl text-center uppercase tracking-widest p-4 pb-12 md:pb-24 text-neutral-400"
```
I just threw in some of the previously used tailwind to get this started. Could you please spruce up to make it look like a true `h1`? I.e. biggest text size, a little grander styling, and some bolder presence.

RWransky reviewed (changes requested):
Just some nitpicks to make this landing page truly next level :) Otherwise, great work!

Commit:
Added ScrollLink, fixed header tags, added logo.

brandondykun commented:
Implemented changes. Also modified the links in the header and footer. There was an about link that linked to the about section (#about) in the header and footer. Since the about section is only on the main / route, I replaced About with Home in the nav sections to link to the main route instead of the About section.

RWransky reviewed (approved):
Thanks for making these changes! 😺
